Session2.Rmd

```diff
@@ -237,27 +238,37 @@ ggplot(data = cycleway_Delft) +
   coord_sf(datum = st_crs(28992))
 ```
 
-### Challenge
+### Challenge (7 minutes)
 
 1. Create a new object that only contains the motorways in Delft. 
 2. How many features does the new object have?
 3. What is the total length of motorways?
 4. Plot the motorways.
 5. Extra: follow the same steps with pedestrian streets. 
 
+Let's first see which value of in the `highway` column holds motorways. There is a value called `motorway`.
+
 ```{r}
 levels(factor(lines_Delft$highway))
 ```
+We extract only the features with the value `motorway`.
 
 ```{r}
 motorway_Delft <- lines_Delft %>% 
   filter(highway == "motorway")
 
-motorway_Delft %>% 
+motorway_Delft
+```
+
+There are `r nrow(motorway_Delft)` features with the value `motorway`.
+
+```{r}
+motorway_Delft_length <- motorway_Delft %>% 
```

Comment:
Something I noticed during the last carpentries was that participants who were not familiar with GIS and R did not really see the link between sf objects in the environment and the maps that were plotted. I think what could be beneficial is that this link between the features and the plotted maps is often highlighted. For instance, when motorways are filtered and plotted, make a brief connection (if you dont do this already) to the variables in the sf-object.

Reply:
Okay, noted, good point! I think this is something we need to do already in the 1st part, when `ggplot2` is introduced and then keep showing that link later between `sf` objects and their visualisation.